classes/static/js/toolbar.js

```diff
@@ -0,0 +1,24 @@
+jQuery( document ).ready(function() {
+	jQuery("body").append('<div id="autoptimize-loader-toolbar"></div>');
```

Comment:
Couldn't we move this inside the click event handler function, just before it is being shown?